### ISSUE TITLE
Add debug level control to admin interface

### ISSUE THREAD

**Issue by adrianhopebailie** [interledgerjs/ilp-connector#446]
It would be great to be able to adjust the global debug/trace level for the application without needing to restart it.

This could be added as an admin API command

**Comment by justmoon** [comment]
Yeah, the debug module actually supports this via `require('debug').enable('ilp*,connector*')`.
